packages/webapp/pages/signup.tsx

```diff
@@ -11,7 +11,7 @@ function Signup(): ReactElement {
   const { formRef } = useAuthForms();
   const router = useRouter();
 
-  const onClose = () => {
+  const onClose = async (): Promise<void> => {
```

Comment:
I think we can revert this change as there seem to be no `await` call was added

Reply:
Fixed ❤️